docs/source/glossary.rst

```diff
@@ -40,6 +40,12 @@ This glossary provides definitions for terms and abbreviations you may encounter
    GPU Acceleration
       The use of GPUs to offload computation-intensive tasks from the CPU, leading to faster processing of tasks like simulations and data analysis.
 
+   GPU-Aware Software
+      Software designed or optimized to utilize GPU acceleration for specific tasks.
+
+   GPU Memory Management
+      Techniques for efficiently managing GPU memory to prevent over-utilization and memory leaks.
```

Comment:
We can show an example here of how to do that. I believe @kevalchheda had helped a user with managing GPU memory for either PyTorch or TF. So that can be shown here as an example.